provider/devices/android.go

```diff
@@ -20,7 +20,14 @@ import (
 func isGadsStreamServiceRunning(device *models.Device) (bool, error) {
 	logger.ProviderLogger.LogInfo("android_device_setup", fmt.Sprintf("Checking if GADS-stream is already running on device `%v`", device.UDID))
 
-	cmd := exec.CommandContext(device.Context, "adb", "-s", device.UDID, "shell", "dumpsys", "activity", "services", "com.shamanec.stream/.ScreenCaptureService")
+	var serviceName = ""
+	if device.UseWebRTCVideo {
+		serviceName = "com.gads.webrtc/.ScreenCaptureService"
+	} else {
+		serviceName = "com.shamanec.stream/.ScreenCaptureService"
+	}
```

Comment:
Consider refactoring this into a separate helper function to avoid code duplication. For example:

```go
func getServiceName(device *models.Device) string {
    if device.UseWebRTCVideo {
        return "com.gads.webrtc/.ScreenCaptureService"
    }
    return "com.shamanec.stream/.ScreenCaptureService"
}
```

The same can be applied fot the others: `packageName` and `activityName`.